### PR TITLE
Adding support for untranslated messages.

### PR DESCRIPTION
**Pull request by AnmAtAnm**
These messages have description comments prefix with {{Notranslate}}.
This will be used to define constants accessible in JSON block definitions. For example, block colors.

### CODE REVIEW

**Comment by rachel-fenichel** [comment]
Please revert msg/qqq.json to avoid reverting this change: https://github.com/google/blockly/commit/b77f8cbebc5cef247116d3df6a428df8addbe53d#diff-6ac46f098be0f96f5b134464dd41eda0

File: i18n/create_messages.py [hunk 1]
```diff
@@ -78,11 +81,20 @@ def main():
   synonym_text = '\n'.join(['Blockly.Msg.{0} = Blockly.Msg.{1};'.format(
       key, synonym_defs[key]) for key in synonym_defs])
 
+  # Read in constants file, which must be output in every language.
```
Comment (rachel-fenichel) [inline i18n/create_messages.py]: Make this a function that returns constants_text.

File: i18n/js_to_json.py [hunk 2]
```diff
@@ -86,14 +90,21 @@ def main():
...
-        result = {}
-        result['meaning'] = match.group(1)
-        result['source'] = match.group(2)
+        key = match.group(1)
+        value = match.group(2)
         if not description:
           print('Warning: No description for ' + result['meaning'])
-        result['description'] = description
+        if (description and _CONSTANT_DESCRIPTION_PATTERN.match(description)):
+          print('Found constant {0} with description: {1}'.
```
Comment (rachel-fenichel) [inline i18n/js_to_json.py]: Remove this print.

File: i18n/create_messages.py (hunk 1 above)
Comment (AnmAtAnm) [inline i18n/create_messages.py]: Done.

File: i18n/js_to_json.py (hunk 2 above)
Comment (AnmAtAnm) [inline i18n/js_to_json.py]: Removed.

**Comment by AnmAtAnm** [comment]
Reverted all qqq.json changes.

**Pull request merged**